From 5e97485b16e290fa620c8131290acbe668d48841 Mon Sep 17 00:00:00 2001
From: Hans Holmberg <hans.holmberg@intel.com>
Date: Fri, 14 Feb 2014 15:29:07 +0100
Subject: [PATCH 09/12] [PORTING_HACK] Android logger: quick fixup of
 compilation error

This fixes a compilation error introduced when porting patch
b7d672d91a39d34753b4ec33b752a38606b82c7f Android logger: support pstore dump

The provided solution needs to be reviewed and integrated into the ported patch.

Change-Id: I04ad5193f443f2ad55e6af3bfead8c10968976d2
Signed-off-by: Hans Holmberg <hans.holmberg@intel.com>
---
 fs/pstore/platform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: intel/fs/pstore/platform.c
===================================================================
--- intel.orig/fs/pstore/platform.c	2014-03-25 19:32:13.050821117 -0700
+++ intel/fs/pstore/platform.c	2014-03-25 19:32:13.047821117 -0700
@@ -378,7 +378,7 @@
 					read = len;
 
 				ret = psinfo->write(PSTORE_TYPE_EXTRA, reason, &id, part,
-						oopscount, read, psinfo);
+						oopscount, read, false, psinfo);
 
 				if (ret < 0)
 					goto extra_done;
